packages/terra-action-footer/src/ActionFooter.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import ActionFooterContainer from './_ActionFooterContainer';
+import styles from './ActionFooter.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Actions to be displayed in the start socket
+   */
+  start: PropTypes.node,
+  /**
+   * Actions to be displayed in the end socket
+   */
+  end: PropTypes.node,
+};
+
+const defaultProps = {
+  start: undefined,
```

Comment:
nit: probably don't **need** default props here, per [this](https://connect.ucern.com/docs/DOC-687470) i'd probably say that it's closest to a primitive? even though it's a node? I'm fine leaving it though

Reply:
Hmm.. am I just missing where the actual resolution on this was stated in your link?

Reply:
Second bullet under 03/02/18 section

Reply:
We discussed default-props for non-required props some more today and landed on  provide default-props for non-required props. However, there was no consensus on what values to use for default-props for non-required props (null/undefined vs empty array/object). For the non-required props for this, I'm cool with using `undefined` as the default prop values.